dropshot/src/lib.rs

```diff
@@ -682,4 +683,9 @@ pub use http::Method;
 
 extern crate dropshot_endpoint;
 pub use dropshot_endpoint::channel;
+pub use dropshot_endpoint::dropshot_server;
 pub use dropshot_endpoint::endpoint;
+
+// Exported for dropshot_endpoint -- not part of the public API.
+#[doc(hidden)]
+pub use trait_variant::make as make_trait_variant;
```

Comment:
Using `trait-variant` temporarily, but it does too much -- we only care about making our endpoints `Send`able. We should copy whatever code from trait-variant we need.

Reply:
No longer use trait-variant, inlining the bits we needed was just around 15 lines of code.